platform/android/src/style/layers/background_layer.cpp

```diff
@@ -26,13 +26,16 @@ namespace android {
     /**
      * Creates a non-owning peer object (for layers currently attached to the map)
      */
-    BackgroundLayer::BackgroundLayer(mbgl::style::BackgroundLayer& coreLayer) : Layer(coreLayer) {}
+    BackgroundLayer::BackgroundLayer(mbgl::style::BackgroundLayer& coreLayer)
```

Comment:
Looks like the generated file changes have not been committed in the past, have we disabled the checks @tmpsantos?
The `.ejs` template file is identical to the version in `galinelle_locationComponent` and `master`.